testing/bloaty/APKBUILD

```diff
@@ -0,0 +1,41 @@
+#-*-mode: Shell-script; coding: utf-8;-*-
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+_commitid=2234386bcee7297dfa1b6d8a5d20f95ea4ed9bb0
+pkgname=bloaty
+# Fake a version until upstream does one
+# format is simply 0.0.0.0.${commit_count}
+pkgver=0.0.0.0.79 
```

Comment:
Sigh, yet another Google project that doesn’t bother to do releases…

For such cases it’s IMO better to use version number in format `0.0_gitYYYYMMDD` (i.e. `0.0_git20161228`) or if you prefer commits count, then `0.0_gitX` (i.e. `0.0_git79`). It makes it more clear that it’s a snapshot from git.